ci/docker/Dockerfile.alpine-release

```diff
@@ -15,37 +12,54 @@ RUN apk add --no-cache \
     fts \
     libevent-dev \
     libtool \
-    libtool \
     linux-pam-dev \
     make \
     musl-dev \
     openssl \
     openssl-dev \
     zlib-dev
 
-# create users and directories
-# note: alpine uses busybox and useradd is not available there
-# also long flags are not available too, so sorry for the
 RUN mkdir -p /var/lib/opensmtpd/empty \
   && adduser _smtpd -h /var/lib/opensmtpd/empty/ -D -H -s /bin/false \
   && adduser _smtpq -h /var/lib/opensmtpd/empty/ -D -H -s /bin/false \
   && mkdir -p /var/spool/smtpd \
   && mkdir -p /var/mail \
   && mkdir -p /etc/mail \
-  && chmod 711 /var/spool/smtpd
+  && chmod 711 /var/spool/smtpd \
+  && mkdir ./install
 
-# Copy contentes of the repo inside the container
-# https://docs.docker.com/engine/reference/builder/#copy
 COPY . /opensmtpd
 
-# build opensmtpd
 RUN ./bootstrap \
   && ./configure \
        --with-gnu-ld \
        --sysconfdir=/etc/mail \
        --with-auth-pam \
   && make \
-  && make install \
-  && cp etc/aliases /etc/mail/aliases
+  && make install
+
+# --- BEGIN Production deployment dockerfile --- #
+
+FROM alpine:3.15 AS deploy
+
+RUN mkdir -p /var/lib/opensmtpd/empty \
+  && adduser _smtpd -h /var/lib/opensmtpd/empty/ -D -H -s /bin/false \
+  && adduser _smtpq -h /var/lib/opensmtpd/empty/ -D -H -s /bin/false \
+  && mkdir -p /var/spool/smtpd \
+  && mkdir -p /var/mail \
+  && mkdir -p /etc/mail \
+  && chmod 711 /var/spool/smtpd
+
+RUN apk add --no-cache libevent \
+    openssl \
+    linux-pam \
+    fts
+
+COPY --from=build /usr/local /usr/local/
```

Comment:
this is too broad, i'd go with something like (not tested yet)

```
COPY --from=build /usr/local/libexec/opensmtpd /usr/local/libexec
COPY --from=build /usr/local/sbin/{smtpctl,smtpd} /usr/local/sbin
COPY --from=build /usr/local/bin/smtp /usr/local/bin
```

Reply:
On it. I dont speak autoconf very well so I couldnt figure out the complete list of the build results

Reply:
Should be resolved.